lib/Elevate/Blockers/Grub2.pm

```diff
@@ -99,6 +103,28 @@ sub _blocker_blscfg ($self) {
     return 0;
 }
 
+sub _blocker_grub_not_installed ($self) {
+
+    return 0 if Cpanel::Pkgr::is_installed('grub2-pc');
+
+    return $self->has_blocker( <<~EOS );
+    The grub2-pc package is not installed. The GRUB2 boot loader is
+    required to upgrade via leapp.
+    EOS
+}
+
+sub _blocker_grub_config_missing ($self) {
+
+    if ( !-e '/boot/grub2/grub.cfg' && !-e '/boot/grub/grub.cfg' ) {
+
+        return $self->has_blocker( <<~EOS );
+        The GRUB2 config file is missing.
```

Comment:
Let's advise them to consider reaching out to support if they hit this blocker.  There is already some wording to do this earlier in this file via `_blocker_grub2_workaround()`